app/serializers/step_serializer.rb

```diff
@@ -99,4 +100,12 @@ def urls
 
     urls_list
   end
+
+  def created_on
```

Comment:
Please stick to the conventions of on/at -> created_**on** is a date, here you are printing a datetime - which should be called created_**at**. Not sure about the text in the translation file, but it would also be appropriate to have it say `"Created at %{datetime} by %{user}"`. I know there are already some inconsistencies with regards to this in our database, but I wouldn't want to add more :)